src/Fmt_ast.ml

```diff
@@ -303,18 +303,61 @@ let rec fmt_longident (li: Longident.t) =
       cbox 0 (fmt_longident li1 $ wrap "(" ")" (fmt_longident li2))
 
 
-let fmt_constant const =
+let fmt_constant (c: Conf.t) const =
+  let is_printable char_code = 0x20 <= char_code && char_code <= 0x7E in
+  let is_not_term_friendly char_code =
+    (* See discussion on PR #70 *)
+    char_code < 4 || char_code = 0x7F
```

Comment:
I don't know where to put the bound (I think escaping only `\x00` would fit me fine).

it would be something like `Char.(' ' <= char)` since Base overrides `<=`. Order on ascii-codes seems more logical to me, since characters are weirdly defined once we also have unicode in mind.

Reply:
Ok, I don't have a strong position, though I think that wrt unicode it's a long way from `char`.